src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SelectionRows/Header/Header.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { styled } from '@mui/material';
+
+import { SelectController } from 'shared/components/FormComponents';
+import { variables } from 'shared/styles';
+
+export const StyledSelectController = styled(SelectController)`
+  .MuiInputBase-root {
+    font-size: ${variables.font.size.md};
+    color: ${variables.palette.primary};
+
+    ::before,
+    ::after {
+      border-bottom: none !important;
```

Comment:
could you pls avoid using important, maybe here you can use '&&'

Reply:
fixed